### PR TITLE
Change include path to match QtC 14

### PR DESCRIPTION
**Pull request by zeule**

### CODE REVIEW

**Comment by CJCombrink** [comment]
Hi @zeule , thanks for the PR

Just to clarify, is this for Qt Creator 14 or 15. I believe 14 is yet to be released, probably not too long from now since it development is in feature freeze.

File: src/spellcheckquickfix.h [hunk 1]
```diff
@@ -20,7 +20,11 @@
 
 #pragma once
 
+#if __has_include(<cppeditor/quickfixes/cppquickfix.h>)
+#include <cppeditor/quickfixes/cppquickfix.h>
```
Comment (CJCombrink) [inline src/spellcheckquickfix.h:24]: Is this due to a header relocation in Qt Creator 14?
At the start of the project I made the call that I will not support past Qt Creator versions, thus if Qt Creator moves the header can you please update the PR to use that header directly, without the additional complexity to work for older versions as well

Comment (zeule) [reply src/spellcheckquickfix.h:23]: Ah, sorry for the mistake. Indeed, the move occurred in the 14th version. I changed the commit message.

**Comment by zeule** [comment]
Ping?

**Comment by CJCombrink** [comment]
> Ping?

Sorry @zeule I have not had any time test.
I have no problem to merge, normally I just do a build test on my side first but this should be fine.

**Comment by zeule** [comment]
Thanks!

**Pull request merged**